src/main/java/hwr/oop/application/console/AddProjectConsole.java

```diff
@@ -0,0 +1,62 @@
+package hwr.oop.application.console;
+
+import hwr.oop.project.Project;
+import hwr.oop.project.ProjectBuilder;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class AddProjectConsole {
+    private Scanner inputStream;
+    private PrintStream outputStream;
+
+    ConsoleColors colors = new ConsoleColors();
+    ProjectBuilder newProject = new ProjectBuilder();
+
+    private String title = "";
+    private String description = "";
+    private LocalDateTime deadline;
+    private static final String FORMAT = "yyyy-MM-dd";
+    private String deadlineString = "";
+
+    public AddProjectConsole(InputStream inputStream, PrintStream outputStream) {
+        this.inputStream = new Scanner(inputStream);
+        this.outputStream = outputStream;
+    }
+
+    public String addNewTitle() {
+        outputStream.println("Please enter the title of the project:");
+        title += inputStream.nextLine();
```

Comment:
```suggestion
        title = inputStream.nextLine();
```
Soll die neue Eingabe an die bereits gespeicherte Variable angehanden werden, oder nicht?

Reply:
Das muss sein, weil es sonst nicht möglich wäre Leerzeichen einzugeben.